Client/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -0,0 +1,534 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import BraveUI
+import CoreData
+import Data
+import Shared
+import BraveShared
+import BraveRewards
+import SnapKit
+
+/// The behavior for sizing sections when the user is in landscape orientation
+enum NTPLandscapeSizingBehavior {
+    /// The section is given half the available space
+    ///
+    /// Layout is decided by device type (iPad vs iPhone)
+    case halfWidth
```

Comment:
This is for iPhone/iPad landscape, where the NTP is half the width of the screen and aligned to the left or right depending on the device (iPad = left, iPhone = right) but Brave Today will not have this same logic (it will just be centered regardless)